OKTA-459954 Embedded SPA + Auth JS for React (#3054)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md

> Before integrating Okta authentication to your React app, you need to register your app in your Okta org. This provides you with the OpenID Connect client ID for authentication requests from your app. Register your app by creating an Okta app integration through the [Okta CLI](https://cli.okta.com/), the [Okta Apps API](/docs/reference/api/apps/), or through the [Admin Console](/docs/guides/quickstart/website/main/#using-the-admin-console) with the following steps.

Reviewer: ```suggestion
Before you integrate Okta authentication into your React app, you need to register your app in your Okta org. This provides you with the OpenID Connect client ID for authentication requests from your app. Register your app by creating an Okta app integration through the [Okta CLI](https://cli.okta.com/), the [Okta Apps API](/docs/reference/api/apps/), or through the [Admin Console](/docs/guides/quickstart/website/main/#using-the-admin-console) with the following steps:
```

1. To create an Okta app integration that represents your React app, sign in to [your Admin Console](https://login.okta.com).
2. From the side navigation, select **Applications** > **Applications**, and then click **Create App Integration**.

> 3. In the dialog box that appears, select **OIDC - OpenID Connect** as the **Sign-on method**, **Single-Page Application** as the **Application type**, and then click **Next**.

Reviewer: ```suggestion
3. In the dialog that appears, select **OIDC - OpenID Connect** as the **Sign-on method**, **Single-Page Application** as the **Application type**, and then click **Next**.
```

4. Fill in the following new app integration settings, and then click **Save**:

    | Setting                | Value/Description                                    |
    | -------------------    | ---------------------------------------------------  |
    | App integration name   | Specify a unique name for your app                  |
    | Grant types            | Select **Authorization Code**, **Interaction Code**, and  **Refresh Token** |

>     | Sign-in redirect URIs  | Specify your app URI for the callback redirect from Okta. For example, `http://localhost:8080/login/callback`. |

Reviewer: ```suggestion
    | Sign-in redirect URIs  | Specify your app URI for the callback redirect from Okta. For example: `http://localhost:8080/login/callback` |
```

>     | Sign-out redirect URIs | Specify your app sign-out redirect URI. For example, `http://localhost:8080`. |

Reviewer: ```suggestion
    | Sign-out redirect URIs | Specify your app sign-out redirect URI. For example: `http://localhost:8080` |
```

Author: I added another sentence after this so retained the period.

>     | Trusted Origins > Base URIs | Specify your app base URI for CORS. For example: `http://localhost:8080`.|

Reviewer: ```suggestion
    | Trusted Origins > Base URIs | Specify your app base URI for CORS. For example: `http://localhost:8080`|
```

Author: Same as above

    | Assignments   | Assign users to your app                                |

>     > **Note:** Cross-Origin Resource Sharing (CORS) is automatically enabled for the Trusted Origins base URI you've specified in the Admin Console. If you're using the [Okta CLI](https://cli.okta.com/manual/apps/create/) to create your SPA Okta app integration, CORS is also automatically enabled for your base URI. You can verify that both **CORS** and **redirect** are enabled for your app by reviewing the **Security** > **API** > **Trusted Origins** page in the Admin Console.

Reviewer: ```suggestion
    > **Note:** Cross-Origin Resource Sharing (CORS) is automatically enabled for the Trusted Origins base URI that you specified in the Admin Console. If you're using the [Okta CLI](https://cli.okta.com/manual/apps/create/) to create your SPA Okta app integration, CORS is also automatically enabled for your base URI. You can verify that both **CORS** and **redirect** are enabled for your app by reviewing the **Security** > **API** > **Trusted Origins** page in the Admin Console.
```
